docs/weaviate/configuration/compression/rq-compression.md

```diff
@@ -117,12 +112,10 @@ RQ can also be enabled for an existing collection by updating the collection def
 
 ## 1-bit RQ
 
-:::caution Preview
-
-**1-bit Rotational quantization (RQ)** was added in **`v1.33`** as a **preview**.<br/>
+:::info Added in `v1.33` and `v1.34`
 
-This means that the feature is still under development and may change in future releases, including potential breaking changes.
-**We do not recommend using this feature in production environments at this time.**
+**1-bit Rotational quantization (RQ)** for the HNSW vector index was added in **`v1.33`**.<br/>
+**1-bit Rotational quantization (RQ)** for the flat vector index was added in **`v1.34`** as a **preview** and should not be used in production.<br/>
```

Comment:
Given this is a warning not to use, imo this could be a separate :::caution::: admonition imho.